apps/frontend/src/components/ShowMembers/ShowMembers.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+
+function App() {
+  const usernames = ["radu", "antonina", "ecem", "maria", "mike", "john"];
+
+  return (
+    <div>
+      <UserCircleStack usernames={usernames} />
+    </div>
+  );
+}
+
+function UserCircleStack({ usernames }) {
+  const displayUsers = usernames.slice(0, 4);
+
+  return (
+    <div className="circle-container">
+      {displayUsers.map((username) => (
+        <UserCircle key={username} username={username} />
+      ))}
+    </div>
+  );
+}
+
+function UserCircle({ username }) {
+  const firstTwoLetters = username.slice(0, 2).toUpperCase();
+
+  return (
+    <div className="circle" style={{ style: "--i: 1" }}>
+      {firstTwoLetters}
+    </div>
+  );
+}
```

Comment:
Usually, the common practice when you work with React is that each component should have its own file, I would recommend that you create a separate folder and file for this component.